src/Okta.Sdk/PoliciesClient.cs

```diff
@@ -12,33 +13,153 @@ namespace Okta.Sdk
     public sealed partial class PoliciesClient : OktaClient, IPoliciesClient
     {
         /// <inheritdoc/>
-        public async Task<T> GetPolicyAsync<T>(string policyId, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<T> GetPolicyAsync<T>(string policyId, CancellationToken cancellationToken = default)
             where T : class, IPolicy
             => await GetPolicyAsync(policyId: policyId, cancellationToken: cancellationToken).ConfigureAwait(false) as T;
 
         /// <inheritdoc/>
-        public async Task<T> CreatePolicyAsync<T>(IPolicy policy, bool? activate, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<T> CreatePolicyAsync<T>(IPolicy policy, bool? activate, CancellationToken cancellationToken = default)
             where T : class, IPolicy
             => await CreatePolicyAsync(policy, activate, cancellationToken).ConfigureAwait(false) as T;
 
         /// <inheritdoc/>
-        public async Task<T> UpdatePolicyAsync<T>(IPolicy policy, string policyId, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<T> UpdatePolicyAsync<T>(IPolicy policy, string policyId, CancellationToken cancellationToken = default)
             where T : class, IPolicy
             => await UpdatePolicyAsync(policy, policyId, cancellationToken).ConfigureAwait(false) as T;
 
         /// <inheritdoc/>
-        public async Task<T> AddPolicyRuleAsync<T>(IPolicyRule policyRule, string policyId, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
We shouldn't remove the existing method, otherwise, it's gonna be a breaking change.

Reply:
This method is not removed, I only added mine before it and simplified the signature:
`public async Task<T> AddPolicyRuleAsync<T>(IPolicyRule policyRule, string policyId, CancellationToken cancellationToken = default)`
instead of
`public async Task<T> AddPolicyRuleAsync<T>(IPolicyRule policyRule, string policyId, CancellationToken cancellationToken = default(CancellationToken ))`

despite it's shown as deleted, it's still there, couple lines further